app/grandchallenge/components/templates/components/partials/civ/image.html

```diff
@@ -9,13 +9,13 @@
         >
         <i class="fas fa-fw fa-download"></i>
         {% if display_inline %}
-            Download {{ civ.interface.title|truncatechars:64 }}:
-            {{ civ.image.name|name|truncatechars:30 }}
+          {{ civ.interface.title|truncatechars:64 }}:
+          {{ civ.image.name|name|stem|truncatechars:32 }}
         {% else %}
-            Download {{ civ.interface.title }}:
-            {{ civ.image.name|stem}}
-            ({{ file.file|suffix }})
```

Comment:
Why did `({{ file.file|suffix }})` disappear?

Why is `|name|stem|` now necessary?